tests/base/quadrature_is_tensor_product.cc

```diff
@@ -26,7 +26,8 @@ template <int dim>
 void
 print_is_tensor_product()
 {
-  Quadrature<dim> q_0(1);
+  Quadrature<dim> q_0(std::vector<Point<dim>>{Point<dim>()},
+                      std::vector<double>{0.});
```

Comment:
```suggestion
  Quadrature<dim> q_0;
```

Reply:
No, not the same. I'm creating a quadrature formula with one point located at the origin and weight zero. You're suggesting to create a quadrature formula with no points.

(I have no idea why the author of the original test chose such a quadrature object, but that's what the test does and I did not feel like investigating whether that's something useful or an accident.)

Reply:
Ah right. I didn't pay enough attention to the initializer lists. Mea culpa.